server/task/task-distributed/src/test/java/org/apache/james/task/eventsourcing/distributed/DistributedTaskManagerTest.java

```diff
@@ -299,4 +301,42 @@ void givenTwoTaskManagerAndATaskRanPerTaskManagerListingThemOnEachShouldShowBoth
                 });
         }
     }
+
+    @Test
+    void givenTwoTaskManagerIfTheFirstOneIsDownTheSecondOneShouldBeAbleToRunTheRemainingTasks(CountDownLatch countDownLatch) throws InterruptedException {
+        try (EventSourcingTaskManager taskManager1 = taskManager()) {
+            try (EventSourcingTaskManager taskManager2 = taskManager(HOSTNAME_2)) {
+                ImmutableBiMap<TaskManager, Hostname> hostNameByTaskManager = ImmutableBiMap.of(taskManager1, HOSTNAME, taskManager2, HOSTNAME_2);
+                TaskId firstTask = taskManager1.submit(new MemoryReferenceTask(() -> {
+                    countDownLatch.await();
+                    return Task.Result.COMPLETED;
+                }));
+
+                awaitUntilTaskHasStatus(firstTask, TaskManager.Status.IN_PROGRESS, taskManager1);
```

Comment:
if you one to know which node is the exclusive consumer, you can call rabbitmqadmin to retrieve the node name

Reply:
I tested the rabbitmqwebadmin and the `consumer_details` field is always an empty list.
Even when calling it via curl to be sure that it was not a problem with the json mapping.

Reply:
In fact it works. Sometimes...

Reply:
As we don't want more flaky tests, we should abstain from using it during the tests. IMHO

Reply:
if you look at the original poc you'll see it just has latency (I did await on the consumer_details)

Reply:
One time i awaited at least 2 minutes ...

Reply:
Furthermore the link between the consumer tag present in the details from the management API to the actual node is not straightforward.

Reply:
weird, I never had this issue. If we can't figure out, that's ok to me